RELEASING.md

```diff
@@ -1,22 +1,29 @@
 ## Releasing a new version
 
-[This GitHub Action](.github/workflows/release.yaml) builds, and releases a new version of the wrapper to both Maven Central and Github whenever a new tag is pushed to the repository.
+[This GitHub Action](.github/workflows/release.yaml) builds, and releases a new version of the wrapper to both Maven Central and Github whenever a new "RC" tag is pushed to the repository.
+
+Release tag format should be `rc<RELEASE VERSION>`.
 
 Full release procedure is as follows:
 
-* Checkout the latest version of the `main` branch.
-* Update wrapper POM version to release (remove -SNAPSHOT)
-* Set wrapper dependency for examples (`examples/build.gradle`) to release version (remove -SNAPSHOT)
-* Update main `README.MD` and `outbound-context-propagation.md` to release version (remove -SNAPSHOT)
-* Commit / push / merge PR
-* Tag that version (eg `v0.0.5` - mind `v` prefix!) with the [annotated tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) and `git push origin <tagname>` 
-* tag will run the release workflow that needs to be *approved* by a maintainer 
-* Update wrapper POM version to next `-SNAPSHOT` version
-* Set wrapper dependency for examples (`examples/build.gradle`) to next `-SNAPSHOT` version
-* Commit / push / merge PR
+* Tag the release candidate version (eg `rc0.0.5`) with the [annotated tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) and `git push origin <tagname>` 
+* Release workflow will be run, it needs to be *approved* by a maintainer 
+* Release will perform following actions:
+  * updates POM / examples / docs version to requested release
+  * commits
```

Comment:
I wasn't sure what "commits" referred to at the first look, how about slightly rewording this list:

```suggestion
  * update POM / examples / docs version to requested release
  * commit changes
```

Reply:
I'll stay with third person, but additional explanation is always welcome.